### PR TITLE
RedHat 8 Compatibility

### PR DESCRIPTION
**Pull request by joschi36**
Ensures Compatibility with RedHat 8 (Tested with AlmaLinux 8)

Ntpd not supported anymore. Now using chrony.

### CODE REVIEW

**Review by josvo — CHANGES_REQUESTED** [review]
Small comments, besides that, LGTM

File: meta/main.yml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+  min_ansible_version: "1.4"
   platforms:
    - name: EL
      versions:
-      - 5
-      - 6
+      - "5"
+      - "6"
+      - "7"
+      - "8"
```
[the remark this answers is not in the document]
Comment (joschi36) [reply meta/main.yml:13]: did not test yet :/

File: meta/main.yml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
    - name: Fedora
      versions:
-      - 16
-      - 17
-      - 18
+      - "16"
+      - "17"
+      - "18"
```
[the remark this answers is not in the document]
Comment (joschi36) [reply meta/main.yml:18]: will delete it

**Review by josvo — APPROVED** [review]
:+1:

**Pull request merged**